testing/path.go

```diff
@@ -209,3 +228,12 @@ func (path *Path) CreateChannels() {
 		panic(err)
 	}
 }
+
+// EnableFeeOnPath enables fee on a channel given a path.
+func EnableFeeOnPath(path *Path) {
```

Comment:
super tiny stylistic nit, but since we probably will use this function in the way we're using it in this test case, could be nice to have it return the modified `Path` so that the usage can be like:

```
path := NewTransferPath(chainA, chainB)
return EnableFeeOnPath(path)
```

Reply:
Done, not sure if you meant just returning or both modifying and returning (so that it can be used in both ways). Let me know if you prefer the former!

Reply:
yes, i meant modifying and returning, exactly as you did thanks so much!